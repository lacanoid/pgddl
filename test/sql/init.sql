CREATE EXTENSION ddlx;
\pset tuples_only

select extname,extversion,nspname,obj_description(e.oid)
  from pg_extension e join pg_namespace n on n.oid=extnamespace
 where extname='ddlx';

