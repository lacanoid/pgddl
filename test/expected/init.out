CREATE EXTENSION ddlx;
\pset tuples_only
\dx ddlx
 ddlx | 0.30    | public | DDL eXtractor functions

